### ISSUE TITLE
crash with Check failed: result.second

### ISSUE THREAD

**Issue by srguiwiz** [nwjs/nw.js#7491]
NWJS Version : 0.46.0
Operating System : CentOS 7.8

### Expected behavior

Our app should keep running as always.

### Actual behavior

Crashes with

```
# Fatal error in , line 0
# Check failed: result.second.
```
and one more line about
```
node_modules/nw/nwjs/lib/libnw.so+0x462f2f8)
```

This seems to have been fixed in Node 14.3.0 and later.  We think NW.js will get a newer Node sooner or later.

A discussion of a fix is at https://github.com/nodejs/node/pull/33321 .

The problem had been discovered in https://github.com/nodejs/node/issues/32463 and https://github.com/nodejs/node/issues/31061 , from Node 14.1.0 onwards.

### How to reproduce

Have not made a small test case from our bigger app.

Hoping NW.js anyway will get a newer Node, and we would report whether the problem is gone.

**Comment by rogerwang** [comment]
Could you please try the latest nightly? http://dl.nwjs.io/live-build/nw46/20200605-233943/d3f126fff/v0.46.2/

**Comment by srguiwiz** [comment]
We have verified this is fixed in 0.46.1, which has Node 14.4.0.
We also have verified this is fixed in 0.46.2.
Thank you.
